resources/test_scenes/test_distant.xml

```diff
@@ -0,0 +1,44 @@
+<scene version="2.0.0">
```

Comment:
This file should be moved to `resources/data/tests/scenes/somedirectory` so it will be picked by the rendergin test suite as described [here](https://mitsuba2.readthedocs.io/en/latest/src/developer_guide/testing.html#rendering-test-suite-and-z-test)

Reply:
Should I fork the data repo as well and create a PR there?

Reply:
Indeed, and have a commit here that updates the submodule.